cask-bootstrap.el

```diff
@@ -30,9 +30,14 @@
 (eval-when-compile
   (defvar cask-directory))
 
+(defconst cask-bootstrap-emacs-version
```

Comment:
Should this perhaps be in `cask.el` called `cask-elpa-dir-version` (or similar). Seems a bit odd that cask.el uses definitions from the bootstrap file (that I see as a self contained file).